db/schema.rb

```diff
@@ -1650,7 +1656,7 @@
     t.boolean  "approved_by_subscriber"
     t.boolean  "approved_by_producer"
     t.datetime "agreed_to_terms_at"
-    t.integer  "susbscriber_id",            limit: 4
```

Comment:
Wait, what?

Reply:
https://github.com/PRX/exchange.prx.org/blob/master/db/migrate/20190710172627_fix_subscriber.rb - looks like the updated schema.rb never showed up over here.